vortex-btrblocks/src/float/stats.rs

```diff
@@ -48,6 +48,7 @@ pub struct FloatStats {
     pub(super) null_count: u32,
     // cache for validity.true_count()
     pub(super) value_count: u32,
+    #[allow(dead_code)]
     pub(super) average_run_length: u32,
```

Comment:
This seems to only be used in tests, so clippy is upset.

Reply:
@AdamGS tried doing this before. The RunEndScheme for floats is unused and if you enable it you degrade performance. I would just delete it and file an issue